package proto_ff;

import "yd_fieldoptions.proto";


message E_UiEffect
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "特效ID"];
}


message Sheet_UiEffect
{
	repeated E_UiEffect E_UiEffect_List = 1[(yd_fieldoptions.field_arysize)=256];
}
